packages/gatsby-theme-project-portal/src/layouts/SearchPageLayout.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FunctionComponent } from "react"
+import { graphql } from "gatsby"
+import SiteSearchWrapper from "../components/SiteSearchWrapper"
```

Comment:
might want to export the SiteSearchWrapper in the ../components/index.tsx file, so you can merge this with the line below.